todo/src/Components/Todos.js

```diff
@@ -0,0 +1,65 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+
+import { addTodo, toggleTodo, deleteTodo } from '../Actions'
+
+
+class Todos extends React.Component {
+    state = {
+        newTodo: ''
+    };
```

Comment:
Since your import statement for React includes the Component object (`import React, { Component }`), you can simply extend the class component directly, like so: `class Todos extends Component`